Add stroke path to Path2D interface per html canvas spec (#350)
jupyter-widgets-contrib/ipycanvas: docs/drawing_paths.rst

Using Path2D
------------

> You can define a Path2D given an SVG path. Note that once the path is created, it is read only, you cannot dynamically change the path value. This means they do not (yet) support Path2D methods like Path2D.move_to, draw_line, etc.

Reviewer: ```suggestion
You can define a Path2D given an SVG path. Note that once the path is created, it is read only, you cannot dynamically change the path value. ``ipycanvas`` does not (yet) support Path2D methods like Path2D.move_to, draw_line, etc.
```
